packages/core/src/workflows/vNext/workflow-next.test.ts

```diff
@@ -12,6 +12,54 @@ import { cloneStep, cloneWorkflow, createStep, createWorkflow } from './workflow
 
 describe('Workflow', () => {
   describe('Basic Workflow Execution', () => {
+    it('should throw error when execution flow not defined', () => {
+      const execute = vi.fn<any>().mockResolvedValue({ result: 'success' });
+      const step1 = createStep({
+        id: 'step1',
+        execute,
+        inputSchema: z.object({}),
+        outputSchema: z.object({ result: z.string() }),
+      });
+
+      const workflow = createWorkflow({
+        id: 'test-workflow',
+        inputSchema: z.object({}),
+        outputSchema: z.object({
+          result: z.string(),
+        }),
+        steps: [step1],
+      });
+
+      expect(() => workflow.createRun()).toThrowError(
+        'Execution flow of workflow is not defined.'
```

Comment:
Let's add a mention how to do that, something like "Add steps to the workflow via .then(), .branch(), etc."